dev/githubanalysis/lib/main.dart

```diff
@@ -273,13 +273,14 @@ Future<int> full(final Directory cache, final GitHub github) async {
       } else {
         priorityResults.openedByNonTeam += 1;
       }
-      if (issue.metadata.isOpen) {
+      // Temporary workaround for https://github.com/SpinlockLabs/github.dart/issues/401
+      if (issue.metadata.isOpen || issue.metadata.state.toUpperCase() == 'OPEN') {
```

Comment:
there's a bunch of other places that use `isOpen`, are they not also affected?

Reply:
Oh! Silly I did not look since it just broke here. I'll check all the others!